### PR TITLE
Feature trn 334 extra nav

### PR DESCRIPTION
**Pull request by nathj07**
From the ticket:

* When looking at the /historical/... info page for a link, the link header itself should lead to the target site and there should be a link to get to the domain info page for that link
* When paging through links in a domain, there should be a link to return to the domain info page

Cover these two points. Tests pass and I have reviewed visually on my dev install. Let me know - particularly if ther eis a visual style gude - I'm no designer and may have violated such rules :smile:

### CODE REVIEW

File: console/templates/historical.tmpl [hunk 1]
```diff
@@ -1,7 +1,8 @@
 
 
  <div class="row" style="width: 90%;">
-        <h2> History for Link {{.LinkTopic}} </h2>
+        <h2>History for Link <a href="{{.LinkTopic}}" target="_blank" title="visit link">{{.LinkTopic}}</a></h2>
+        <h3><a href="/links/{{.Domain}}" title="view domain info">Domain Info</a></h3>
```
Comment (pchristopher1275) [inline console/templates/historical.tmpl]: Generally in our code base, the word Domain means Top-Level-Domain-Plus-1 (see url.go). The link here points at /links/{{.Domain}} but Domain (above in controllers) is defined as u.Host. In order to use it as you've done, it must be u.ToplevelDomainPlusOne(). The net result is that clicking on the link that reads "Domain Info" results in a 500 error.

**Comment by pchristopher1275** [comment]
May wanna merge master into this branch. [When I downloaded this branch, I `git merge master` and got a delta, so i'm assuming there have been patches merged into master since this branch was forked].

File: console/test/controllers_test.go [hunk 2]
```diff
@@ -663,6 +680,20 @@ func TestListHistorical(t *testing.T) {
 		}
 	})
 
+	doc.Find(".container h3 a").First().Each(func(index int, sel *goquery.Selection) {
+		text := strings.TrimSpace(sel.Text())
+		if !strings.HasPrefix(text, "Domain Info") {
+			t.Fatalf("[.container h3 a] Bad prefix got %s, expected 'Domain Info'", text)
```
Comment (pchristopher1275) [inline console/test/controllers_test.go]: No problem with this: just a comment for anyone that doesn't know. Go's printf interpreter recognizes the flag %q which prints a (possibly escaped) quoted string (i.e. "foo" not foo). When reporting errors in tests, I use it to highlight the string, and make it easier to see embedded spaces, and the empty string.

Comment (dankinder) [inline console/test/controllers_test.go]: Cool that's useful, this sounds like a good case for that.

**Comment by nathj07** [comment]
So I may the merge of `master` and now the full test suite fails on 'TestUpdateDomain`. Does anyone know about this already or shall I go ahead and try to fix that?

File: console/templates/historical.tmpl (hunk 1 above)
Comment (nathj07) [inline console/templates/historical.tmpl]: Understood. Will change the code accordingly.

**Comment by nathj07** [comment]
Ok, so further thoughts?

I've also added the final point on this ticket:
* When the user adds a new link, we currently see a notice that it was successful; we should redirect the user to the /historical/... info page for that link (and preferably still show a success message)

Which has been modified to:
* One alternative would be to change the success message so that it lists each of the added links such that they link to the relevant historical page

I've run the full test suite on this and they all pass.

File: console/templates/historical.tmpl (hunk 1 above)
Comment (dankinder) [inline console/templates/historical.tmpl]: @nathj07 is there still a change to make here?

**Comment by nathj07** [comment]
Discussed the `{{.Domain}}` thing with @dankinder on chat, all is well there.
Sorted the use of `%q` think this is right now.

**Comment by dankinder** [comment]
:facepunch:

**Pull request merged**